lib/Helpers.js

```diff
@@ -539,10 +539,7 @@ class Helpers {
           bulkBody.push(actionBody, payloadBody);
         } else if (operation === 'update') {
           actionBody = serializer.serialize(action);
-          payloadBody =
-            typeof chunk === 'string'
-              ? `{"doc":${chunk}}`
-              : serializer.serialize({ doc: chunk, ...payload });
```

Comment:
Is this the reason why this breaks the bulk operation for regular use of this helper method?

(Also sorry the comment mentioning me got lost in the mail)

Reply:
@nhtruong 
Yes. The old method only supported doc, so we modified it to support script as well, but that will not work with the previous method of input.

Reply:
We should avoid breaking changes to existing users. Unless there's no other way around this?